lib/geodb.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+var geodb = require('fxa-geodb')()
+var ACCURACY_MAX_KM = 200
+var ACCURACY_MIN_KM = 25
+
+/**
+* Thin wrapper around geodb, to help log the accuracy
+* and catch errors. On success, returns an object with
+* `location` data. On failure, returns an empty object
+**/
+module.exports = function (log) {
+  return function (ip, request) {
```

Comment:
Per [**eslint-config-fxa**/index.js:41](https://github.com/pdehaan/eslint-config-fxa/blob/0238e80bbb7fc0be93cab9f2a7a6ee71b504cac4/index.js#L41), we're not checking for unused function arguments:

```js
'no-unused-vars': [2, {'vars': 'all', 'args': 'none'}]
```